### PR TITLE
FAI-503: Score Cards: Support use of underscore ("_") in predicate field names

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.redhat.com/browse/FAI-503

I widened the regex to include all non-white space characters (as the PMML [specification](http://dmg.org/pmml/v4-4/GeneralStructure.html#xsdType_FIELD-NAME) supports _anything_).

### CODE REVIEW

**Review by tiagobento — APPROVED** [review]

**Comment by tiagobento** [comment]
@kelvah @sjamboro @ederign Hi... so I jumped the gun and merged this PR. Sorry about not waiting for your review. Please take your time to take a look, we won't release anything without your approval. I'm sure if you find anything @manstis will be more than happy to send a new PR 😝

**Comment by GitHubCuCu** [comment]
thx!!!  when can I get this new release in Vscode ?

File: packages/pmml-editor/src/editor/components/DataDictionary/DataTypeItem/DataTypeItem.tsx [hunk 1]
```diff
@@ -309,12 +311,10 @@ const DataTypeItem = (props: DataTypeItemProps) => {
         <section
           className={"editable-item__inner"}
           tabIndex={0}
-          onClick={handleEditStatus}
+          onClick={(event) => handleEditStatus(event)}
```
Comment (kelvah) [inline packages/pmml-editor/src/editor/components/DataDictionary/DataTypeItem/DataTypeItem.tsx:314]: hi @manstis, it's not necessary to wrap the callback inside an arrow function here. `handleEditStatus` will get `event` as parameter anyway. It's a minor optimization, I would probably avoid another PR just for this.

**Review by kelvah — COMMENTED** [review]
LGTM :+1:

File: packages/pmml-editor/src/editor/components/DataDictionary/DataTypeItem/DataTypeItem.tsx (hunk 1 above)
Comment (manstis) [reply packages/pmml-editor/src/editor/components/DataDictionary/DataTypeItem/DataTypeItem.tsx:314]: Agreed.. the dangers of switching from Java to Typescript :-)

**Pull request merged**